client/util/util.go

```diff
@@ -20,7 +20,7 @@ type EnterpriseCertificateConfig struct {
 
 // Libs specifies the locations of helper libraries.
 type Libs struct {
-	SignerBinary string `json:"signer_binary"`
+	SignerBinary string `json:"ecp"`
```

Comment:
nit: maybe rename SignerBinary to ECP (btw golang convention for acronyms is all caps)